e2e-tests/stream-mode-tests.sh

```diff
@@ -65,20 +61,19 @@ if ! [[ $(command -v jq) ]]; then
     exit 1
 fi
 cd "${test_dir}"
-source ./e2e-tests/common.sh
-test "?family=Clive" "?subject.given=Cliff" "?subject.given=Cliff"
+
+validate_counts "?family=Clive" "?subject.given=Cliff" "?subject.given=Cliff"
 
 if [[ ${total_patients_streamed} == ${total_test_patients}
       && ${total_encounters_streamed} == ${total_test_encounters}
       && $total_obs_streamed == ${total_test_obs} ]] ;
 then
-  print_message "STREAMING MODE WITH PARQUET SYNC EXECUTED SUCCESSFULLY"
+  print_message "STREAMING MODE WITH PARQUET SINK EXECUTED SUCCESSFULLY"
 else
-  print_message "STREAMING MODE WITH PARQUET SYNC TEST FAILED"
+  print_message "STREAMING MODE WITH PARQUET SINK TEST FAILED"
   exit 1
 fi
 
-cd "${test_dir}"
 mkdir fhir
```

Comment:
Where is this `fhir` directory used?

Reply:
its used for sinking in the patient,encounter and obs as seen below